src/services/services.js

```diff
@@ -0,0 +1,67 @@
+import firebase, { db } from './firebase';
+
+export const isCorrectEmail = (email) => {
+  // eslint-disable-next-line no-useless-escape
+  const regExp = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  return regExp.test(email);
+};
+
+export const isCorrectPassword = (password) => {
+  // eslint-disable-next-line no-useless-escape
+  if (password.length < 8) {
+    return false;
+  }
+  return true;
```

Comment:
U can just write `return length > 8`

Reply:
thx for the advice, i  did it.
```suggestion
  password.length >= 8
```